nipype/interfaces/bids_utils.py

```diff
@@ -123,13 +107,17 @@ def __init__(self, infields=None, **kwargs):
 
         # used for mandatory inputs check
         undefined_traits = {}
-        for key in infields:
+        for key in infields or []:
```

Comment:
this should be done in list outputs as well.

and travis tests are not testing the `have_bids==True` branch.

Reply:
@satra this might be required here, since it's using `**kwargs` and setting traits